### ISSUE TITLE
Enhancement: Listen Only for Migrated Tokens

### ISSUE THREAD

**Issue by creosmart** [chainstacklabs/pump-fun-bot#22]
Hi everyone,

I would like to create a listener similar to the one we already have, but specifically for tokens that migrated from pump.fun to Raydium.

I realized that the program ID for the migration process is `39azUYFWPz3VHgKCf3VChUwbpURdCHRxjWVowf5jUJjg`, but it seems I'm unable to receive the output correctly, as we do with the existing listener. Could this issue be related to the IDL? Any help would be highly appreciated.

This is the example output I would like to receive, but only for the migrated tokens:
```
  "name": "No risk No rich",
  "symbol": "NR",
  "uri": "https://ipfs.io/ipfs/QmdTDZAk9rYsHXsdVKv1MUHLd1fjvRetdtAKJefYuWshmp",
  "mint": "G5cCvrdfj1HeyhZmbjn1WHAx2fDzdaE2fUoCsBddpump",
  "bondingCurve": "5v53sTCUGXRWveLc7RF9S9SkzY1kEwxH73YszAPeQQT4",
  "associatedBondingCurve": "5bRWieUZjGfFAFMi6t2Y9n1eXmtFsKGavTcQ2SZcHr5",     
  "user": "9vn3RFsnEeB4yKfMu84Z1haUxpqu8Ef6K9cxBmru6mHp"
```

**Comment by akegaviar** [comment]
Hey there, check this one out [Solana: Listening to pump.fun migrations to Raydium](https://docs.chainstack.com/docs/solana-listening-to-pumpfun-migrations-to-raydium).

Also, starting from the next week (**week of March 10**) we'll be rolling out updates to the bot v2 based on the feedback and the reported issues, including updating to the latest libs, better error handling etc. We are already actively working on it.

That'll be in a separate branch.

Check out the readme for the roadmap that I just added in the README file for the full overview.

**Comment by smypmsa** [comment]
Hi @creosmart! Please check the refactored branch, there is an example of listening to migrations and getting required addresses.

https://github.com/chainstacklabs/pump-fun-bot/tree/refactored/main-v2

**Comment by smypmsa** [comment]
Yes, it works on my end. Please compare it to the transactions you see on solscan:
https://solscan.io/account/39azUYFWPz3VHgKCf3VChUwbpURdCHRxjWVowf5jUJjg

**Comment by creosmart** [comment]
Hi Anton,
I found the reason why I'm not receiving the information: the block payload is too large (>1MB). As a result, the WebSocket connection is automatically closed. Is there a workaround for this issue?
`websockets.exceptions.ConnectionClosedError:   sent 1009 (message too big) ... exceeds limit of 1048576 bytes
`

**Comment by smypmsa** [comment]
@creosmart please have a look at updated examples:
https://github.com/chainstacklabs/pump-fun-bot/tree/main/learning-examples/listen-migrations

There are a few types of listeners for migrations. If you encounter the issue again, feel free to reopen it.
